spec/rubocop/cop/rails/uniq_before_pluck_spec.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::UniqBeforePluck do
+  let(:corrected) { 'Model.uniq.pluck(:id)' }
+  subject(:cop) { described_class.new }
+
+  shared_examples_for 'UniqBeforePluck cop' do |source, line_no = 1|
```

Comment:
Seems to me the line check is a bit redundant, given the simplicity of a selector's position check.